mlflow/server/js/src/model-registry/components/ModelView.js

```diff
@@ -271,3 +335,12 @@ export class ModelView extends React.Component {
     );
   }
 }
+
+const mapStateToProps = (state, ownProps) => {
+  const modelName = ownProps.model.name;
+  const tags = getRegisteredModelTags(modelName, state);
+  return { ...ownProps, tags };
```

Comment:
```suggestion
  return { tags };
```

I think `ownProps` here can be removed.

Reply:
Agreed, my mistake.